README.rst

```diff
@@ -5,8 +5,8 @@ RNCryptor-python
     :alt: Actual PyPI version
     :target: https://pypi.python.org/pypi/rncryptor/
 
-.. image:: https://travis-ci.org/RNCryptor/RNCryptor-python.svg?branch=master
```

Comment:
Shouldn't it be reverted back?

Reply:
Easy peasy.

Change done in commit 2300f5b.